### ISSUE TITLE
Allow people to see the path of etheral jaunting wizards

### ISSUE THREAD

**Issue by Notamaniac** [yogstation13/yogstation-classic#1041]
One of the main issues with wizard is ethereal jaunt, which essentially allows you to always escape any situation at fullspeed or ambush people in an instants notice without them being able to react.

With both other teleport escape spells, they both at least have SOME drawback  (Teleport results in you screaming out your destination, and blink results in smoke and not knowing where the hell you'll end up), but the only 'drawback' on jaunt is the single second you have to spend reforming when you've finished your jaunt.

So basically, to stop this becoming an 'always pick' spell, it should be nerfed leave a visible aftertrail showing the exact position of the wizard, so it stops being a get out of jail free card, allows people to actually chase after you, and stops wizzies from ambushing people without some forewarning

**Comment by ToGWtF** [comment]
jaunt is fine imo.  it has a delay when reappearing and any shooter worth their shot can hit them as they land

**Comment by Notamaniac** [comment]
That delay is less than a second, and they can't be hit while they're in the reappearing phase (or at least it's not long enough for anyone to react fast enough to shoot them). And even then, all the wizard has to do is move as soon as they can and they'll probably dodge whatever is being thrown at them. 

Not only that, but the wizard is free to cast whatever barrage of spells they want when they finish jaunting, so even if the shooter is fast enough to react to shoot them, all it takes is for the wizard to cast either magic missile, lighting bolt, or timestop, and they can buy enough time to either shake off the stun, or retreat if they've been injured severely (keep in mind wizard robes have some degree of armour, so it takes more than five shots to down them).

**Comment by FluffySurvivor** [comment]
There is no need to nerf wiz. Especially since jaunt has a long cooldown which you should abuse. 

Wiz needs a rework.

**Comment by ShadowDeath6** [comment]
The only problem is that people who are standing around are usually preoccupied with something, like a sleeper or defibbing someone, so wiz can just run up to them, magic misdle, and ei nath for le fair wizard fun meemee

**Comment by Stealthkibbler** [comment]
If you taze a wizard the stun carries on through his Jaunt, and you can taze him mid jaunt animation which causes him to be unable to move during his whole jaunt.

**Comment by Axemeaquestion** [comment]
Still salty about that round shadow?

**Comment by ShadowDeath6** [comment]
yes
